apps/voting/test/contracts/voting_delegation.js

```diff
@@ -439,8 +438,8 @@ contract('Voting delegation', ([_, root, voter, anotherVoter, thirdVoter, repres
       })
 
       context('when the input is not valid', () => {
-        const repeat = (x, y) => [...Array(x).map(() => y)]
-        const voters = repeat(101, voter)
+        const repeat = (x, y) => [...Array(x)].map(() => y)
+        const voters = repeat(71, voter)
```

Comment:
We could define this as a constant somewhere in these tests, or even better, add a getter to the mock contract that returns the constant and fetch it from there.